Gruntfile.js

```diff
@@ -31,50 +37,57 @@ module.exports = function(grunt) {
       }
     },
 
-    babel: {
-      options: {
-        sourceMap: true,
-        presets:  ['env'],
-        plugins: ['transform-object-rest-spread']
-      },
-      dist: {
-        files: [{
-          cwd: 'src',
-          expand: true,
-          src: ['**/*.js'],
-          dest: 'dist',
-          ext:'.js'
-        }]
-      },
-      distTestNoSystemJs: {
-        files: [{
-          cwd: 'src',
-          expand: true,
-          src: ['**/*.js'],
-          dest: 'dist/test',
-          ext:'.js'
-        }]
-      },
-      distTestsSpecsNoSystemJs: {
-        files: [{
-          expand: true,
-          cwd: 'spec',
-          src: ['**/*.js'],
-          dest: 'dist/test/spec',
-          ext:'.js'
-        }]
+    ts: {
+      build: {
+        tsconfig: 'tsconfig.json',
+        src: ["src/**/*.ts", "spec/**/*.ts"],
+        outDir: "dist"
       }
     },
 
+    // babel: {
+    //   options: {
+    //     sourceMap: true,
+    //     presets: ['env'],
+    //     plugins: ['transform-object-rest-spread']
+    //   },
+    //   distTestNoSystemJs: {
+    //     files: [{
+    //       cwd: 'src',
+    //       expand: true,
+    //       src: ['**/*.js'],
+    //       dest: 'dist/test',
+    //       ext: '.js'
+    //     }]
+    //   },
+    //   distTestsSpecsNoSystemJs: {
+    //     files: [{
+    //       expand: true,
+    //       cwd: 'spec',
+    //       src: ['**/*.js'],
+    //       dest: 'dist/test/spec',
+    //       ext: '.js'
+    //     }]
+    //   }
+    // },
+
     mochaTest: {
       test: {
         options: {
           reporter: 'spec'
         },
-        src: ['dist/test/spec/test-main.js', 'dist/test/spec/*_spec.js']
+        src: ['dist/spec/*_spec.js']
       }
     }
   });
 
-  grunt.registerTask('default', ['clean', 'copy:src_to_dist', 'copy:pluginDef', 'babel', 'mochaTest']);
+  grunt.registerTask('default', [
+    'clean', 
+    'copy:src_to_dist', 
+    'copy:pluginDef', 
+    'ts:build', 
+    'copy:dist_src', 
+    // 'babel',
```

Comment:
Why aren't we using babel anymore? I also think this might be the reason of failed tests.

Reply:
The reason seems to be the changed output format. Grafana has a template plugin with ts that seems to work without babel. It does not have tests though.